batch-job.go

```diff
@@ -194,11 +205,13 @@ const BatchJobExpireTemplate = `expire:
 
 // BatchJobResult returned by StartBatchJob
 type BatchJobResult struct {
-	ID      string        `json:"id"`
-	Type    BatchJobType  `json:"type"`
-	User    string        `json:"user,omitempty"`
-	Started time.Time     `json:"started"`
-	Elapsed time.Duration `json:"elapsed,omitempty"`
+	ID      string             `json:"id"`
+	Type    BatchJobType       `json:"type"`
+	User    string             `json:"user,omitempty"`
+	Started time.Time          `json:"started"`
+	Elapsed time.Duration      `json:"elapsed,omitempty"`
+	Status  BatchJobStatusType `json:"status"`
+	Message *string            `json:"message,omitempty"`
```

Comment:
```suggestion
	Message string            `json:"message,omitempty"`
```

Doesn't need to be a pointer to be empty.

Reply:
and also what is this Message supposed to be anyway a comment would help?

Reply:
if its Error then we can simply have the Message renamed as `Error string` we use this commonly everywhere.

Reply:
@vadmeste Please comment on your plans for this new field.